### PR TITLE
PrettyPets-Bot by Karoline Mann and Birgit Nehrwein

### PR DESCRIPTION
**Pull request by nehrwein**
The project was to build a chatbot using javascript. This was the first pairprogramming assignment (Karoline Mann and Birgit Nehrwein). We decided to build a chatbot for a pet grooming company.

Check out our project here: https://pretty-petts-karo-birgit.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -30,18 +33,138 @@ const showMessage = (message, sender) => {
...
+// The Bot starts here
+//The greeting function sends a message as the bot asking for your name.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello and welcome to Pretty Pets! <br> <br>What's the name of your furry friend?`, 'bot')
+}
+
+// The user enters dog name in the handleInput function
+// There's a condition-check to make sure that the user enters a valid name
```
Comment (Rephili) [inline code/script.js:43]: Good job! We tried it and it works. /Rebecca and Darya

File: README.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We started by doing a flowchart to visualize the conversation flow between the bot and the user. After that we started to work with the given starter code in Javascript following the iterations in the project brief guidelines.
+In the js code we worked with 
+- DOM selectors
+- global variables
+- functions
+- eventhandlers
+- conditional statements
+
+If we would have more time we would get more elaborated on the styling and responsivness.
```
Comment (DALA746) [inline README.md:15]: Great job with flowchart! Very useful for future projects! // Darya & Rebecca

File: code/script.js [hunk 3]
```diff
@@ -30,18 +33,138 @@ const showMessage = (message, sender) => {
...
+  document
+  .getElementById('treatmentBtn')
+  .addEventListener('click', () => viewTreatments())
+
+  document
+  .getElementById('appointmentBtn')
+  .addEventListener('click', () => bookTreatment())
+}
+
+// User choose View treatment and two new buttons gets displayed (Big Lebowski and Michelle Pfieffer)
```
Comment (Rephili) [inline code/script.js:78]: Great that you added comments in the code. Makes it easy to follow. :) /Darya and Rebecca

File: code/script.js [hunk 4]
```diff
@@ -30,18 +33,138 @@ const showMessage = (message, sender) => {
...
+  // After 1 second, show the next question by invoking the next function passing the name into it to have access to the user's name if we want to use it in the next question from the bot.
+  setTimeout(() => nameOfPet(userInput), 1000)
+}
+
+// Questions from the Bot about the user selection and creating two buttons (Book appointment and View treamtment)
+const nameOfPet = () => {
+  showMessage(`Would you like to book an appointment or look at our treatments?`, 'bot')
+  
+  inputWrapper.innerHTML = `
+  <button id="appointmentBtn">Book Appointment</button>
```
Comment (DALA746) [inline code/script.js:66]: Good naming of the id!

**Review by DALA746 — COMMENTED** [review]
You have created a very readable and "easy to understand" code. Everything is working as it should and the code is following the general and blue requirements! Looking forward to see more of your projects! Great job! // Darya & Rebecca

**Pull request closed without merging**